lib/BridgeAbstract.php

```diff
@@ -142,7 +155,10 @@ public function setDatas(array $inputs){
 			if($time !== false
 			&& (time() - static::CACHE_TIMEOUT < $time)
 			&& (!defined('DEBUG') || DEBUG !== true)){
-				$this->items = $this->cache->loadData();
+				$cached = array();
```

Comment:
I mean creating ``$cached`` and resetting its value the line after. You could remove this line, no ?